recipes/quota_anchor/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "quota_anchor" %}
+{% set version = "0.0.1a0" %}
+{% set tag = "v0.0.1a0" %}
```

Comment:
It might be slightly easier to put the `v{{ version }}` in the url rather than setting tag here. Just one fewer line to have to update in the future.

Reply:
Thanks, I have made the modification accordingly.